Dockerfile

```diff
@@ -1,34 +1,59 @@
 # https://docs.docker.com/reference/builder/
 # NOTE dockerignore is ignored https://github.com/docker/docker/issues/9455
 
-FROM ensime/ensime:v1.x
-MAINTAINER Sam Halliday, sam.halliday@gmail.com
+FROM debian:jessie
```

Comment:
Well, in that case, I could just use that one instead :)  Coursier cache and java was all I needed just now, but I guess later I'll try to have at least smoketests for the whole stack like Emacs - you do sbt gen-ensime and everything on small projects, no?